securitas/templates/login.html

```diff
@@ -0,0 +1,26 @@
+{% extends "with_navbar.html" %}
+{% block title %}Self-Service Portal{% endblock %}
```

Comment:
Perhaps use a more descriptive title that mentions "login" somewhere?

Reply:
Will do.